package.json

```diff
@@ -45,9 +46,9 @@
     "@cerner/terra-cli": "^1.6.0",
     "@cerner/terra-dev-site": "^8.0.0",
     "@cerner/terra-enzyme-intl": "4.6.0",
-    "@cerner/terra-functional-testing": "^2.0.0",
+    "@cerner/terra-functional-testing": "^2.2.0",
     "@cerner/terra-open-source-scripts": "^1.0.1",
-    "@cerner/webpack-config-terra": "^1.3.1",
+    "@cerner/webpack-config-terra": "file:./cerner-webpack-config-terra-1.3.1.tgz",
```

Comment:
Same here it would be easier to make relevant changes on dependent projects if the latest version of webpack-config was used here.

Reply:
Ok, thx!